ci/docker-pull-or-build-and-push.sh

```diff
@@ -7,6 +7,13 @@ DIRECTORY_HASH=$(git rev-parse HEAD:ci/$image_name)
 export DIRECTORY_HASH
 cd ci/$image_name/
 
+USER_ID=3000
+export USER_ID
```

Comment:
Nit: no need to split assignment and `export` in case of trivial (non-failable) right-hand sides of the assignment like `3000`. The entire reason for splitting assignment and export is https://github.com/codacy/codacy-hadolint/blob/master/codacy-hadolint/docs/description/SC2155.md